src/DI/AwsSdkNetteExtension.php

```diff
@@ -19,7 +19,8 @@ class AwsSdkNetteExtension extends Nette\DI\CompilerExtension
 		'credentials' => [
 			'key' => NULL,
 			'secret' => NULL
-		]
+		],
+        'use_aws_shared_config_files' => true
```

Comment:
Could you please change your indentation to tabs?

Reply:
I changed it, is it okay now?